src/lms-ds-loader/edfi_lms_ds_loader/loader_facade.py

```diff
@@ -12,30 +12,37 @@
 
 from edfi_lms_ds_loader.helpers.constants import Table
 from edfi_lms_ds_loader.helpers.argparser import MainArguments
-from edfi_lms_ds_loader.migrator import migrate
+from edfi_lms_ds_loader import migrator
 from edfi_lms_file_utils import file_reader
-from edfi_lms_ds_loader.df_to_db import upload_file
+from edfi_lms_ds_loader import df_to_db
 from edfi_lms_ds_loader.mssql_lms_operations import MssqlLmsOperations
 
 logger = logging.getLogger(__name__)
 
 
 def _load_users(csv_path: str, db_adapter: MssqlLmsOperations) -> None:
```

Comment:
```suggestion
@catch_exceptions
def _load_users(csv_path: str, db_adapter: MssqlLmsOperations) -> None:
        users = file_reader.get_all_users(csv_path)
        df_to_db.upload_file(db_adapter, users, Table.USER)
```

Reply:
I will try it out...

Reply:
the [@catch_exceptions](https://github.com/Ed-Fi-Exchange-OSS/LMS-Toolkit/blob/main/src/extractor-lib/edfi_lms_extractor_lib/helpers/decorators.py) will log the message and return true or false depending on the resolution of the callable.

Reply:
For a long time I thought it was failing in a surprising way, but I finally figured out that I was making a pytest mistake:

Wrong
```
           def __raise(csv_path):
                raise Exception("bad things")

            mocker.patch("edfi_lms_file_utils.file_reader.get_all_users", return_value=__raise)
```

Right:
```
           def __raise(csv_path):
                raise Exception("bad things")

            mocker.patch("edfi_lms_file_utils.file_reader.get_all_users", side_effect=__raise)
```

(use of `side_effect` instead of `return_value`)